### DISCUSSION TITLE
How to make a submodule checkout to a commit at the update moment??

### DISCUSSION THREAD

**Discussion by JLuisAlcaraz** [git-lfs/git-lfs#5274] [Q&A]
Hi all!,
I'm making a repository with a submodule linked to it.
When the repository is cloned, I init the submodule and update it.
The problem is it points always to the last commit of the submodule repo.

I can make the update and then checkout to other commit, but I want to do this at the "update" moment.

Is it possible??

**Comment by bk2204** [comment]
Hey,

This is not really a question about Git LFS, but rather a question about Git in general.  I'd recommend sending an email [to the Git mailing list](https://git-scm.com/community) and ask there, since I'm not really sure how to achieve what you want.

**Reply by JLuisAlcaraz** [reply]
You're right, sorry.
Thank you very much.
